benchmarks/bench_renorm.py

```diff
@@ -0,0 +1,119 @@
+import torch
+from triton.testing import do_bench
+
+import flashinfer
+
+
+def normal_distribution(std):
+    def normal_noise(shape, device):
+        return torch.randn(shape, device=device) * std
+
+    normal_noise.__name__ = f"normal_distribution(std={std})"
+    return normal_noise
+
+
+def gumbel_distribution(beta):
+    def gumbel_noise(shape, device):
+        U = torch.rand(shape, device=device)
+        eps = 1e-20
+        return torch.log(-torch.log(U + eps) + eps) / beta
+
+    gumbel_noise.__name__ = f"gumbel_distribution(beta={beta})"
+    return gumbel_noise
+
+
+def init_seed_top_p_renorm(*args, **kwargs):
+    torch.manual_seed(42)
```

Comment:
We don't really to fix seed here because there is no randomness in renorm kernels. Please move them to the front of `main()` to fix `distrib`.